### PR TITLE
Make compile successful 

### PR DESCRIPTION
**Pull request by ChenaLee**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by khanhntd — APPROVED** [review]
LGTM.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,4 @@
-## My Project
-
-TODO: Fill this README out!
+## Amazon CloudWatch Agent Test
```
Comment (SaxyPandaBear) [inline README.md:1]: I'd prefer if this title was more descriptive. The format that I've always stuck to for README files is:
```
Title of the repo/project
------------------------
Short tag line or elevator pitch for the purpose of the repo. 

## Subheader
Stuff

...
```

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,39 @@
+module github.com/aws/amazon-cloudwatch-agent-test
+
+replace github.com/influxdata/telegraf => github.com/aws/telegraf v0.10.2-0.20220502160831-c20ebe67c5ef
```
Comment (SaxyPandaBear) [inline go.mod:3]: Do we need telegraf here?

File: go.mod [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+go 1.18
+
+require (
+	github.com/aws/aws-sdk-go v1.44.118
```
Comment (SaxyPandaBear) [inline go.mod:8]: The integration tests don't **need** the v1 SDK. Can you remove this and update imports as necessary? I think there are imports in the tests that use stuff like `aws.String()` but import from the v1 SDK when there are equivalent functions in v2. My e2e mem test PR included a change with that. Can follow that as a guide.

**Review by SaxyPandaBear — APPROVED** [review]
LGTM. Minor comments that don't need to be addressed immediately in this PR.

Follow up actions we need to take (not necessarily in this order):
1. Port over the GitHub Actions workflows for integration testing
2. Set up CI checks (build, lint, etc) and automate execution of them in PRs
3. Maybe CODEOWNERS to automate assigned reviewers for PRs
4. Figure out our strategy for when these integration tests will execute (we should NOT run them automatically for all PRs all the time because that allows people to exhaust our resources)

File: go.mod (hunk 3 above)
Comment (ChenaLee) [reply go.mod:8]: oh this might have been before I removed the dependency to cwa main repo. will retry

File: go.mod (hunk 2 above)
Comment (ChenaLee) [reply go.mod:3]: same as below

Comment (ChenaLee) [reply go.mod:3]: yeah looks like it works now without telegraf. With agent repo dependency this had to be added

File: go.mod (hunk 3 above)
Comment (ChenaLee) [reply go.mod:8]: sure I can update

**Pull request merged**